### ISSUE TITLE
Installation fails with "syntax error, unexpected '$kebabName' (T_VARIABLE), expecting ']'"

### ISSUE THREAD

**Issue by joshea0** [Laravel-Backpack/Generators#93]
# Bug report

### What I did:
Ran `composer require backpack/generators --dev` as part of [the installation instructions](https://backpackforlaravel.com/docs/4.1/installation)

### What I expected to happen:
The package installs without error.

### What happened:
Received error:
```
syntax error, unexpected '$kebabName' (T_VARIABLE), expecting ']'
```

Full command output:

```
$ composer require backpack/generators --dev
Using version ^3.1 for backpack/generators
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing backpack/generators (v3.1.2): Loading from cache
Writing lock file
Generating optimized autoload files
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   ParseError 

  syntax error, unexpected '$kebabName' (T_VARIABLE), expecting ']'

  at vendor/backpack/generators/src/Console/Commands/ChartBackpackCommand.php:41
    37|         echo Artisan::output();
    38| 
    39|         // Create the chart route
    40|         Artisan::call('backpack:add-custom-route', [
  > 41|             'code' => "Route::get('charts/".$kebabName."', 'Charts\\".$studlyName."ChartController@response')->name('charts.'."$kebabName".'.index');",
    42|         ]);
    43|         echo Artisan::output();
    44|     }
    45| }

      +1 vendor frames 
  2   [internal]:0
      Composer\Autoload\ClassLoader::loadClass("Backpack\Generators\Console\Commands\ChartBackpackCommand")

  3   [internal]:0
      spl_autoload_call("Backpack\Generators\Console\Commands\ChartBackpackCommand")
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

Installation failed, reverting ./composer.json to its original content.
```

### What I've already tried to fix it:
I created a fresh laravel app (using `composer ...`) and tried to only install backpack following [the installation instructions](https://backpackforlaravel.com/docs/4.1/installation). I got the same error on the fresh install.

### Backpack, Laravel, PHP, DB version:

- backpack/crud: `4.1.11`
- backpack/generator: `v3.1.2`
- Laravel: `v7.16.1`
- PHP `7.3.17`
- DB: `5.7.30`

---

~Working on this now. I believe I found the fix, so I should have a PR open later today.~

Seems someone else is working on it already: https://github.com/Laravel-Backpack/Generators/pull/92

**Comment by joshea0** [comment]
I see a PR was opened to fix this already: https://github.com/Laravel-Backpack/Generators/pull/92

I won't submit my own PR since @iMokhles seems to be working on it.

**Comment by joshea0** [comment]
I think https://github.com/Laravel-Backpack/Generators/pull/90 is perhaps fixing this issue as well.

I apologize for not checking if there were open PRs for this before opening this issue. I checked to see if there were any existing issues, but forgot to check for relevant PRs. Feel free to close this if you'd prefer.

**Comment by tabacitu** [comment]
Thanks a lot for reporting this @joshea0 ! Indeed, we've made a bit of a blunder. It's now fixed since #92 , so version 3.1.3 should install nicely for you. 

A `composer install --dev backpack/generators` should get the proper (fixed) version now (3.1.3).  If not, please do `composer install --dev backpack/generators:"^3.1.3"`, that should force composer to pick up at least the fixed version.

I'm sorry for the downtime - hopefully you'll win the time back in a few more weeks, when we launch the UI for this package, for free 😉 Stick around, I think you'll love it!

> I apologize for not checking if there were open PRs for this before opening this issue. I checked to see if there were any existing issues, but forgot to check for relevant PRs. Feel free to close this if you'd prefer.

No worries. It's better to have more issues about it than _no issue_ about it 😄 

Cheers!

**Comment by joshea0** [comment]
No problem, we all make blunders 😄 I got it working just fine by dropping my version down to 3.1.1 - I'll upgrade to 3.1.3 tomorrow.

No worries about the downtime. I'm pretty sure I've won the time back several times over already from how much time I've saved using backpack up to this point 😄

Thanks for the awesome work you and all the contributors do on backpack! I'm excited to see the UI you're launching soon.
